05debianlatest_python_flask/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM debian:latest
+
+MAINTAINER Eugene Naryshkin <aurcame@gmail.com>
+
+RUN apt-get update && apt-get install -y python python-pip wget && pip install Flask
```

Comment:
Would be good to add here a Flash application to start as base image starts 'init' process that won't use the 'Flash' python method.